action-ssh-to-runner/action.yaml

```diff
@@ -46,12 +50,21 @@ runs:
       run: |
         TMATE_SOCK=/tmp/tmate.sock
         echo "TMATE_SOCK=$TMATE_SOCK" >> $GITHUB_ENV
-        nohup tmate -f ${{ env.TMATE_CONF }} -S ${TMATE_SOCK} -F > /dev/null &
-        echo "$!" > /tmp/tmate.pid
-        # We need to wait a bit for tmate to start.
-        sleep 5
-        tmate -f ${{ env.TMATE_CONF }} -S ${TMATE_SOCK} wait tmate-ready
+        if [[ "${{ inputs.detached }}" == "true" ]]; then
+          nohup tmate -f ${{ env.TMATE_CONF }} -S ${TMATE_SOCK} -F > /dev/null &
+          # Wait briefly for tmate to start and retrieve connection info
+          sleep 5
+          TMATE_SSH=$(tmate -f $TMATE_CONF -S ${TMATE_SOCK} display -p '#{tmate_ssh}')
```

Comment:
minor: 
There is a bit of repetition with the next step (getting connection info and printing the message).

You could maybe refactor the next step to have 2 possible messages depending on the value of `inputs.detached`, and then also exit the while loop after printing the message if `inputs.detached` is true

Reply:
you are right, will handle that. thanks for catching it

Reply:
I updated [here](https://github.com/scality/actions/pull/154/commits/41746c9b036e4d2958e2de1a0004a78931ddf748). I don't want to check every time in while loop, so chose to exit earlier instead.